### PR TITLE
add 404 page and route

### PR DESCRIPTION
**Pull request by harsilspatel**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -118,3 +118,7 @@ app.get('/server/status', (req, res) => {
   const outputJson = { status: 'True' };
   res.status(200).send(outputJson);
 });
+
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', '/client', '/page_404.html'));
```
Comment (khanguslee) [inline server/server.js]: Don't need the slashes before client and page_404.html.

File: client/page_404.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <img height=50% width=50%
+     src="https://illustatus.herokuapp.com/?title=Oops,%20page%20not%20found&fill=%234f86ed"/>
```
Comment (khanguslee) [inline client/page_404.html]: This won't work offline.

Create your own 404 page instead?

**Review by khanguslee — CHANGES_REQUESTED** [review]
404 page is definitely useful! Might be more suitable to have the 404 page use the sidebar and navbar templates so that users can easily go back to an endpoint that works.

File: server/server.js (hunk 1 above)
Comment (harsilspatel) [reply server/server.js]: oh right, fixing it.

File: client/page_404.ejs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <link rel="stylesheet" type="text/css" href="css/bootstrap.css">
+      <link rel="stylesheet" type="text/css" href="css/index.css">
+      <link rel="stylesheet" type="text/css" href="css/power-calibration.css">
+  </head>
+  <body>
+      <% include ./partials/header %>
+
+      <% include ./partials/sidebar %>
+    <img height=50% width=50%
+    <!-- credits: https://github.com/blairlee227/IlluStatus -->
```
Comment (khanguslee) [inline client/page_404.ejs]: This breaks the page. Move comment one line above.

File: server/server.js [hunk 4]
```diff
@@ -118,3 +118,12 @@ app.get('/server/status', (req, res) => {
   const outputJson = { status: 'True' };
   res.status(200).send(outputJson);
 });
+
+app.get('*', (req, res) => {
+  res.render(path.join(__dirname, '..', 'client', 'page_404'), {
+    header: 'Oops...',
+    // eslint-disable-next-line object-shorthand
```
Comment (khanguslee) [inline server/server.js]: Don't think we should add anymore eslint-disable lines unless we have a specific reason.

File: server/server.js [hunk 5]
```diff
@@ -118,3 +118,12 @@ app.get('/server/status', (req, res) => {
...
+    sidebar: sidebar,
```
Comment (khanguslee) [inline server/server.js]: line should just be `sidebar,`  to not have the eslint error to occur. New in ES6: See https://alligator.io/js/object-property-shorthand-es6/

**Review by khanguslee — CHANGES_REQUESTED** [review]

**Review by khanguslee — CHANGES_REQUESTED** [review]
Thanks for fixing the issue! I'm gonna be a pain in the ass, but can you set the Oops... as a title of the page and leave the header as MHP DaShboard. I swear it's the last thing 💃

**Pull request merged**